cs/v1/CS_v1.0.0.yaml

```diff
@@ -1625,53 +1513,27 @@ components:
           $ref: '#/components/schemas/Barge'
     TruckTransport:
       title: Truck Transport
-      x-stoplight:
-        id: wk5bv2x6qf7r8
       type: object
       description: Transport mode Truck.
-      required:
-        - modeOfTransportType
-      properties:
-        modeOfTransportType:
-          x-stoplight:
-            id: zta37a83ipubo
-          enum:
-            - TRUCK
-          description: The mode of transport as defined by DCSA. For the Truck Transport mode this needs to be TRUCK.
-        licensePlate:
-          type: string
-          x-stoplight:
-            id: 70ltreyqhy5tu
-          description: 'A license plate is a tag that is attached to a vehicle and displays a unique number or code assigned to the vehicle. The format, design, and issuing authority for license plates vary by country, state, and province.'
-          maxLength: 15
-        chasisLicensePlate:
-          type: string
-          x-stoplight:
-            id: yl83mtk9qd9j7
-          description: A chassis number is a unique identifying number or code assigned to the chassis of a vehicle.It may also be referred to as a "vehicle identification number" (VIN) or "frame number.
-          maxLength: 15
-    CombinedTransport:
-      title: Combined Transport
-      x-stoplight:
-        id: 9epz3n74vupyr
+    OtherTransport:
+      title: Other Transport
       type: object
-      description: Combined Transport mode.
+      description: Other Transport modes.
       required:
-        - modeOfTransportType
+        - modeOfTransport
       properties:
-        modeOfTransportType:
-          x-stoplight:
-            id: qezsxdt0llmco
+        modeOfTransport:
+          type: string
           enum:
             - RAIL_TRUCK
             - BARGE_TRUCK
             - BARGE_RAIL
             - MULTIMODAL
-          description: "The mode of transport as defined by DCSA. For the Combined Transport mode this needs to be one of the below options.\r\n\r\n- `RAIL_TRUCK` (When rail and truck are expected to be the mode of transport in a leg of a proposed routing)\r\n- `BARGE_TRUCK`(When barge and truck are expected to be the mode of transport in a leg of a proposed routing)\r\n- `BARGE_RAIL`(When barge and rail are expected to be the mode of transport in a leg of a proposed routing)\r\n- `MULTIMODAL`(When mode of transport is not really defined or unknown at this stage)"
+            - RAIL
```

Comment:
I thought `TRUCK` and `RAIL` were covered by objects - and not enum?

Reply:
This is for the other proposal.